### DISCUSSION TITLE
Shall we participate in Summer-2022?

### DISCUSSION THREAD

**Discussion by tyn1998** [hypertrons/hypertrons-crx#313] [General]
Guys, Summer-2022 is comming soon and they will start to accept proposals from cummunities on Mar 28. Shall we participate in this year?
@will-ww @frank-zsy @zhuxiangning  @LiuChangFreeman  @heming6666  

Links:
- https://mp.weixin.qq.com/s/7N-2hGXZG7DWiG1F5xsjkQ
- https://summer-ospp.ac.cn/

Timeline:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/32434520/161472231-c6e0d1a9-edb7-47a1-b39b-aa7f3d737376.png">

**Comment by frank-zsy** [comment]
Definitely yes, I think hypertrons-crx is quite ready to let other developers join in now.

**Reply by tyn1998** [reply]
Hi, Frank~ I notice that the event announcement happened in [hypertrons/hypertrons](https://github.com/hypertrons/hypertrons/tree/master/docs/event) rather than hypertrons-crx last two years. Should we move all our event-related discussion to hypertrons/hypertrons? 

Discussion transfer is supported in GitHub:
<img width="267" alt="image" src="https://user-images.githubusercontent.com/32434520/161366302-f4ec8a98-93c4-4428-b9fd-b721c9e68a16.png">
Discussion feature should be enabled in hypertrons/hypertrons first.

**Comment by will-ww** [comment]
Yes, just go~

**Comment by tyn1998** [comment]
Copy that!

**Comment by TieWay59** [comment]
Hello folks, where can I find further info about specific tasks? 👂

**Reply by zhuxiangning** [reply]
Possible tasks are listed in this [discussion](https://github.com/hypertrons/hypertrons-crx/discussions/321) and new ideas are welcome💡
